proposed/phpdoc-meta.md

```diff
@@ -7,19 +7,19 @@ at the smallest possible level of detail. This type of documentation aids contri
 code to, for example, understand what type of information needs to be passed to specific methods, or how to be able to
 consume a class of the project that a consumer want to use.
 
-By documenting specific elements inside the source code the documentation for that part of the source code will be less
+By documenting specific elements inside the source code, the documentation for that part of the source code will be less
 susceptible to becoming out of date.
 
-PHPDoc as a notation has existed for more than ten years now, is heavily inspired by JavaDoc, and is currently in use by a
-significant percentage of public PHP projects in the field.
+PHPDoc as a notation has existed for more than fifteen years now, is heavily inspired by JavaDoc, 
```

Comment:
Ulf Wendel presented phpdoc and its notation in October 2000. In case you want to be more exact than "more than fifteen years" ;-)

Reply:
I was wondering about that... my only mental timeline was knowing roughly when Josh had it and Greg joined in :-D

Reply:
relative time information without a clear reference date ages badly - maybe mention a year instead of "x years ago" which will be outdated next year...

Reply:
Yep, that's what I'll be doing now that I have the actual landmark.